.github/workflows/srtool.yml

```diff
@@ -1,6 +1,18 @@
 name: Srtool build
 
-on: push
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
Is there a specific pattern of tab we could match on rather than any tag?

Reply:
I checked the existing tag~s~ (https://github.com/paritytech/cumulus/tags) and for now, the list is too short for a pattern. Since the frequency of new tags is still low, I think picking up all tags is better than every single and fair for now. My proposal would be to refine the pattern as soon as we start using tags , that would trigger the runtime build but that we also want to exclude.